Pipfile

```diff
@@ -6,51 +6,43 @@ name = "pypi"
 [packages]
 dataclasses = "*"
 python-dateutil = "*"
```

Comment:
Since arxiv-browse does not produce a package to upload to pypi I didn't convert to Poetry. 

I relaxed the dependency constraints.